protected/humhub/modules/admin/views/group/edit.php

```diff
@@ -1,17 +1,17 @@
 <?php
 
 use humhub\modules\ui\form\widgets\SortOrderField;
-use yii\widgets\ActiveForm;
+use humhub\modules\ui\form\widgets\ActiveForm;
 use yii\helpers\Url;
 use yii\helpers\Html;
-use humhub\compat\CHtml;
 use humhub\modules\user\widgets\UserPickerField;
 use humhub\modules\space\widgets\SpacePickerField;
+
 ?>
 
-<?php $this->beginContent('@admin/views/group/_manageLayout.php', ['group' => $group]) ?>
+<?php $this->beginContent('@admin/views/group/_manageLayout.php', ['group' => $group, 'acknowledge' => true]) ?>
```

Comment:
Why not? I think that user can leave unsaved changes on the page and move out of the page

Reply:
I agree with @MasterHans

Reply:
@yurabakhtin Are you sure? This line introduces the variable `acknowledge` in this view 
https://github.com/humhub/humhub/blob/29df6ab189f83b7c554a3b7fa3308085e3215bdd/protected/humhub/modules/admin/views/group/_manageLayout.php#L8 without using it.

Reply:
I was wrong, I have reviewed and got we should pass the param into the `ActiveForm` instead of `beginContent` as it is already implemented in the latest version of this PR, so this conversation can be resolved.